README.md

```diff
@@ -151,13 +151,13 @@ everything else from the plugin:
 <Chain "FilterOutDetailedDockerStats">
   <Rule "CpuUsage">
     <Match "regex">
-      Type "^cpu.usage$"
+      Type "^cpu.percent$"
     </Match>
     Target "return"
   </Rule>
   <Rule "MemoryUsage">
     <Match "regex">
-      Type "^memory.usage$"
```

Comment:
if this rule is changed to work only with memory.percent then the rule name should probably change to MemoryPercent.